MaraBot/Core/Weekly.cs

```diff
@@ -10,13 +10,15 @@ public class Weekly
         public string PresetName;
         public string Seed;
         public Dictionary<string, TimeSpan> Leaderboard;
+        public DateTime GeneratedAt;
```

Comment:
How about reusing `Timestamp` here for more consistency throughout the bot?

Reply:
Didn't know we were using timestamp already. I only saw DateTime.Now being used so I assumed we were using DateTime.

Reply:
I was referring to the name of the property. It's already called `Timestamp` in the embed:
https://dsharpplus.github.io/api/DSharpPlus.Entities.DiscordEmbed.html?q=Timestamp#DSharpPlus_Entities_DiscordEmbed_Timestamp

Reply:
Ah I see now. That's fine by me to rename GeneratedAt to Timestamp.